### PR TITLE
Iteration three transaction

### PR DESCRIPTION
**Pull request by CaptainSpectacular**
Transaction and TransactionRepository classes made

### CODE REVIEW

File: lib/data_analyst.rb [hunk 1]
```diff
@@ -46,5 +46,17 @@ def self.find_customers(file)
      csv['last_name'],
      csv['created_at'],
      csv['updated_at']].transpose
-  end 
+
```
[the remark this answers is not in the document]
Comment (CaptainSpectacular) [reply lib/data_analyst.rb]: fixed

File: lib/transaction_repository.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  end
+
+  def all
+    @transactions
+  end
+
+  def find_by_id(id)
+    all.detect { |transaction| transaction.id == id }
+  end
+
```
[the remark this answers is not in the document]
Comment (CaptainSpectacular) [reply lib/transaction_repository.rb]: fixed

File: lib/data_analyst.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (CaptainSpectacular) [reply lib/data_analyst.rb]: fixed

File: lib/transaction_repository.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (CaptainSpectacular) [reply lib/transaction_repository.rb]: fixed

File: lib/transaction_repository.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
+require_relative 'data_analyst'
+require_relative 'transaction'
+# This is the TransactionRepository class
+class TransactionRepository
+  def initialize
+    @transactions = []
+  end
+
+  def from_csv(csv)
```
[the remark this answers is not in the document]
Comment (agpiermarini) [reply lib/transaction_repository.rb:9]: Will refactor later if necessary.

[the remark this answers is not in the document]
Comment (agpiermarini) [reply lib/transaction_repository.rb:9]: Will refactor later if necessary.

**Pull request merged**